awsagentprovider/src/test/java/software/amazon/opentelemetry/javaagent/providers/AwsMetricAttributeGeneratorTest.java

```diff
@@ -499,14 +499,31 @@ public void testRemoteAttributesCombinations() {
     mockAttribute(NET_SOCK_PEER_ADDR, null);
     mockAttribute(NET_SOCK_PEER_PORT, null);
 
-    // Validate behavior of Remote Operation from HttpTarget - with 1st api part, then remove it
+    // Validate behavior of Remote Operation from HttpTarget - with 1st api part. Also validates
+    // that RemoteService is extracted from HttpUrl.
     mockAttribute(HTTP_URL, "http://www.example.com/payment/123");
-    validateExpectedRemoteAttributes(UNKNOWN_REMOTE_SERVICE, "/payment");
+    validateExpectedRemoteAttributes("www.example.com", "/payment");
     mockAttribute(HTTP_URL, null);
 
-    // Validate behavior of Remote Operation from HttpTarget - without 1st api part, then remove it
+    // Validate behavior of Remote Operation from HttpTarget - with 1st api part. Also validates
+    // that RemoteService is extracted from HttpUrl.
     mockAttribute(HTTP_URL, "http://www.example.com");
-    validateExpectedRemoteAttributes(UNKNOWN_REMOTE_SERVICE, "/");
+    validateExpectedRemoteAttributes("www.example.com", "/");
+    mockAttribute(HTTP_URL, null);
+
+    // Validate behavior of Remote Service from HttpUrl
+    mockAttribute(HTTP_URL, "http://192.168.1.1:8000");
+    validateExpectedRemoteAttributes("192.168.1.1:8000", "/");
+    mockAttribute(HTTP_URL, null);
```

Comment:
Lets add one test for `192.168.1.1` (e.g. no port)

Reply:
done